testutil/common/tester.go

```diff
@@ -0,0 +1,310 @@
+package common
+
+import (
+	"context"
+	"strconv"
+	"testing"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/common/types"
+	testkeeper "github.com/lavanet/lava/testutil/keeper"
+	planstypes "github.com/lavanet/lava/x/plans/types"
+	projectstypes "github.com/lavanet/lava/x/projects/types"
+	spectypes "github.com/lavanet/lava/x/spec/types"
+	subscriptionkeeper "github.com/lavanet/lava/x/subscription/keeper"
+	subscriptiontypes "github.com/lavanet/lava/x/subscription/types"
+)
+
+type Tester struct {
+	T *testing.T
+
+	GoCtx   context.Context
```

Comment:
are the 2 contexts synchronized?

Reply:
yes: all the Advance{Something})() methods update both of them.